debian/prebuild.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -exu
```

Comment:
We're using bash here, so nothing blocks us from adding `-o pipefail`. It would be nice, because the script uses pipes.

Reply:
That's the only commit I have and it is very minor. I'll just close it.